Gemfile

```diff
@@ -27,6 +27,8 @@ gem 'pundit', '~> 2.1'
 gem 'puma', '~> 5.0'
 # Use SCSS for stylesheets
 gem 'sass-rails', '>= 6'
+# An utility first CSS framework
+gem 'tailwindcss-rails', '~> 0.3.0'
```

Comment:
I think the gem is not needed if we are going to use tailwind via webpacker, we just need the npm module, right?

Reply:
oh I guess you ran https://github.com/rails/tailwindcss-rails/pull/16/files#diff-5aa2349060350e9685318fe05e3403f899f77fb6e74fce0c6aee323aadbaf9b8 . I'm good then